src/Games/brain-even.js

```diff
@@ -0,0 +1,24 @@
+import getRandomNumber from '../randomCount.js';
+import gamesCore from '../index.js';
+
+const descriptionOfGame = 'Answer "yes" if the number is even, otherwise answer "no".';
+
+const isEven = (checkNumber) => {
```

Comment:
`const isEven = n => n % 2 === 0;` ?

Reply:
ow, yes, that really beauty,
& here is no need '=== true' ))
fixed